### DISCUSSION TITLE
just saving csv files

### DISCUSSION THREAD

**Discussion by AKHCE** [MilesCranmer/PySR#455] [Q&A] [answered]
Hi,
Passing equation_file to PySRRegressor enables us to save the results in three formats .csv, .csv.bkup, and .pkl.  I am just wondering if there is a quick setting to tell PySRRegressor just to save .csv files as this is the one I personally use often.
Thank you so much for your support,
Cheers

**Comment by MilesCranmer** [comment, marked as answer]
Hi @AKHCE,

Right now, no, all the files are saved automatically. It might be easier to just delete them afterwards, as it doesn't affect performance at all since those files are so small. Regarding the default behavior, I think it's better to have saving be opt-out rather than opt-in so that new users don't accidentally lose a long run. 

But I'm open to adding an option to disable it though!

The .csv.bkup version is so that if the process is quit while Julia is writing the .csv version, progress is not lost. 

Cheers,
Miles

**Reply by AKHCE** [reply]
Thanks for your reply.
